components/admin_console/license_settings/license_settings.jsx

```diff
@@ -405,6 +408,9 @@ export default class LicenseSettings extends React.PureComponent {
                             className='form-horizontal'
                             role='form'
                         >
+                            <div className='license-expir-ing-ed-notice'>
```

Comment:
Where do we define this style?

Reply:
this is a left-behind code from the initial design that needs to be removed.